README.md

```diff
@@ -338,6 +338,8 @@ Mistake triggers are specified for individual fights in the [ui/oopsyraidsy/data
 
 ![oopsy screenshot](screenshots/promo_oopsy.png)
 
+You can copy oopsy logs to clipboard by clicking it. (You have to uncheck `Enable Clickthrough` checkbox from the overlay option.)
```

Comment:
```suggestion
You can copy oopsy lines to the clipboard by clicking them.
(You may need to uncheck `Enable Clickthrough` checkbox from the OverlayPlugin option.)
```

Some small edits, and also adding a line break to follow https://sembr.org/.

Reply:
Oh... I didn't know sembr. I'll review again with that.